state-chain/custom-rpc/src/lib.rs

```diff
@@ -371,10 +379,48 @@ mod boost_pool_details {
 			}
 		}
 	}
+
+	#[derive(Serialize, Deserialize)]
+	struct PendingFees {
+		deposit_id: PrewitnessedDepositId,
+		fees: Vec<AccountAndAmount>,
+	}
+
+	#[derive(Serialize, Deserialize)]
+	pub struct BoostPoolFeesRpc {
+		fee_tier: u16,
+		#[serde(flatten)]
+		asset: Asset,
+		pending_fees: Vec<PendingFees>,
+	}
+
+	impl BoostPoolFeesRpc {
+		pub fn new(asset: Asset, fee_tier: u16, details: BoostPoolDetails) -> Self {
+			BoostPoolFeesRpc {
+				fee_tier,
+				asset,
+				pending_fees: details
+					.pending_boosts
+					.into_iter()
+					.map(|(deposit_id, owed_amounts)| PendingFees {
+						deposit_id,
+						fees: owed_amounts
+							.into_iter()
+							.map(|(account_id, amount)| AccountAndAmount {
+								account_id,
+								amount: NumberOrHex::from(amount.fee),
```

Comment:
In the LP API we're moving towards using U256 instead of NumberOrHex - we probably want to be consistent here too. I think there was another place in another boost PR we used NumberOrHex too, so would check all of them.